lib/protoflow-blocks/doc/sys/read_socket.mmd

```diff
@@ -0,0 +1,9 @@
+block-beta
+    columns 4
+    ReadSocket space:2 Sink
+    ReadSocket-- "output" -->Sink
+
+    classDef block height:48px,padding:8px;
+    classDef hidden visibility:none;
+    class ReadSocket block
+    class Sink hidden
```

Comment:
Make sure to newline-terminate all text files

Reply:
I don't see any warnings in github now, fixed